mne/viz/raw.py

```diff
@@ -834,7 +834,10 @@ def _prepare_mne_browse_raw(params, title, bgcolor, color, bad_color, inds,
         size = tuple([float(s) for s in size])
 
     fig = figure_nobar(facecolor=bgcolor, figsize=size)
-    fig.canvas.set_window_title(title)
+    if title:
+        fig.canvas.set_window_title(title)
+    else:
+        fig.canvas.set_window_title("Raw")
```

Comment:
I guess you can also do

`fig.canvas.set_window_title(title if title else "Raw")`

Reply:
Oh yes. Fixed.